cmake/CompilerWarnings.cmake

```diff
@@ -1,9 +1,10 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright 2013-2022 The Foundry Visionmongers Ltd
 
+# Set platform compiler to have relevent warnings and checks enabled.
 function(set_default_compiler_warnings target_name)
 
-    set(CLANG_WARNINGS
+    set(clang_warnings
```

Comment:
[question] How come no leading `_`?  Is it because it has function scope?

Reply:
Yeah.
`variable` is for local scope vars, whilst `_variable` is for what cmake-lint calls "private directory variables"

I guess this is the distinction between private and "really private" that you were upholding before.